python/lsst/ts/standardscripts/scheduler/testutils/scheduler_controller.py

```diff
@@ -0,0 +1,140 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["SchedulerController"]
+
+import asyncio
+
+from lsst.ts import salobj
+
+
+class SchedulerController(salobj.Controller):
```

Comment:
I strongly encourage renaming this to MockScheduler because the current class name sounds like something that *controls* the scheduler, rather than something that pretends to *be* a Scheduler